src/loaders/spritesheetParser.js

```diff
@@ -23,10 +24,17 @@ module.exports = function ()
             metadata: resource.metadata.imageMetadata
         };
 
+        // Prepend url path unless the resource image is a data url
+        if (Resource.isDataUrl(resource.url)) {
```

Comment:
Did you try this? `Resource.isDataUrl` is a boolean property, not a function.

https://github.com/englercj/resource-loader/blob/ba23abc5fd4662136aacee40cab2e565a611086a/src/Resource.js#L52